bundles/binding/org.openhab.binding.cardio2e/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,29 @@
+Manifest-Version: 1.0
+Private-Package: org.openhab.binding.cardio2e.internal
+Ignore-Package: org.openhab.binding.cardio2e.internal
+Bundle-License: http://www.eclipse.org/legal/epl-v10.html
+Bundle-Name: openHAB Cardio2e Binding
+Bundle-SymbolicName: org.openhab.binding.cardio2e
+Bundle-Vendor: openHAB.org
+Bundle-Version: 1.7.0.qualifier
+Bundle-ManifestVersion: 2
+Bundle-Description: This is the Cardio2e binding of the open Home Aut
+ omation Bus (openHAB)
+Import-Package: org.apache.commons.lang,
+ org.openhab.core.binding,
+ org.openhab.core.events,
+ org.openhab.core.items,
+ org.openhab.core.library.items,
+ org.openhab.core.library.types,
+ org.openhab.core.types,
+ org.openhab.model.item.binding,
+ org.osgi.framework,
+ org.osgi.service.cm;version="1.4.0",
+ org.osgi.service.component,
+ org.osgi.service.event,
+ org.slf4j
+Export-Package: org.openhab.binding.cardio2e
+Bundle-DocURL: http://www.openhab.org
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
```

Comment:
That's a valid point, the code in this repo should indeed stay Java7 compatible.